src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -572,32 +577,40 @@ To get the job result(s), the user has to use the `#%get_result()` pragma by pro
 #%get_result(name=JOB_NAME)
 ```
 
-or by the job id:
+Or by the job ID:
```

Comment:
previously u used "Or," which I approve but here u removed it. I prefer with "comma"